File: app/src/main/java/com/kkkkkn/readbooks/model/entity/AnimationConfig.java
package com.kkkkkn.readbooks.model.entity;

import android.view.View;

import java.io.Serializable;

public class AnimationConfig implements Serializable {
    public View view;
    public float moveX;
    public float moveY;
    public float scaleX;
    public float scaleY;


    public boolean isEmpty() {
        return view == null
                || moveX == 0
                || moveY == 0
                || scaleX == 0
                || scaleY == 0;
    }

}

File: app/src/main/java/com/kkkkkn/readbooks/model/entity/AnimationConfig.kt
package com.kkkkkn.readbooks.model.entity

import android.view.View
import java.io.Serializable

class AnimationConfig : Serializable {
    var view: View? = null
    var moveX = 0f
    var moveY = 0f
    var scaleX = 0f
    var scaleY = 0f
    val isEmpty: Boolean
        get() = view == null || moveX == 0f || moveY == 0f || scaleX == 0f || scaleY == 0f
}